package com.TesteEtagio.Teste4.exceptions;

public class ResourceNotFoundException {
}
